lib/state.coffee

```diff
@@ -73,3 +114,11 @@ state.first = ->
   for urlPage, idx in firstUrlPages when urlPage not in oldPages
     link.createPage(urlPage, firstUrlLocs[idx]) unless urlPage is ''
 
+state.syncDomWithLocation = ->
+  main = ""
+  for {site, slug} in state.fromLocation(location)
+    dataSite = ""
+    if site != "view"
+      dataSite = """data-site="#{site}" """
+    main += """<div id="#{slug}" #{dataSite}class="page"></div>\n"""
+  $('section.main').html(main)
```

Comment:
This function is where we introduce more confusion to the bootstrapping of wiki-client.

We replace the contents of `<section class="main">` with our own page divs computed from `window.location`.

For comparison, here is where @paul90 made a similar intervention in the DAT/hypercore variant.
https://github.com/paul90/wiki-client-dat-variant/commit/15cb65667861abd9c9af80d7970b43f1c001246b#diff-5b6ac0ea3ef5483a6e071c1ff85d99a3c4ef31d97b567deff46686fb93a25921R17-R33

For the time being, we are deferring the work to understand and change the existing wiki-client bootstrap.